docs/cli_reference/dfget_gen-doc.md

```diff
@@ -17,6 +17,14 @@ dfget gen-doc [flags]
   -p, --path string   destination path of generated markdown documents (default "/tmp")
 ```
 
+### Options inherited from parent commands
+
+```
+      --alivetime duration    server will stop if there is no uploading task within this duration (default 5m0s)
```

Comment:
I do not think this is correct, since persistent flags will be inherited from the parent command to all child commands. If the flags is owned by child command, then just keep them  in the child command. @lowzj

Reply:
Because that the `dfget server` is launched by `dfget` automatically, user may want to control the alive time and expire time when they just use `dfget` to download files.